tests/test_yaml_generators/test_yaml_oci_generator.py

```diff
@@ -65,13 +67,13 @@ def test_generate_tags(self, mock_choice):
         """Test label string generator."""
 
         dc = self.yg.default_config()
-        for key in self.module.TAG_KEYS.keys():
+        for key in OCI_TAGS:
```

Comment:
If `OCI_TAGS` is changed to a dataclass.
```suggestion
        for key in (field.name for field in dataclasses.fields(OCITags)):
```